Gemfile

```diff
@@ -18,4 +18,5 @@ group :development do
   gem "puppet-blacksmith"
   gem "nokogiri", "~> 1.6.0"
   gem "guard-rake"
+  gem "xmlrpc" if RUBY_VERSION >= '2.3'
```

Comment:
locally I'm running the tests against ruby `2.4.1p111` and in order to get the tests to pass you need to install this gem. of course when your ruby is < `2.3` the builds will barf apparently.